<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="Prp-Test-01" tests="1" errors="0" failures="1" skipped="0" time="8.289" timestamp="2023-11-14T12:09:59.480000">
<testcase classname="Prp-Test-01" name="Open_Application" time="8.171">
<failure message="MaxRetryError: HTTPConnectionPool(host='127.0.0.1', port=4724): Max retries exceeded with url: /session (Caused by NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x000001ED92652690&gt;: Failed to establish a new connection: [WinError 10061] Es konnte keine Verbindung hergestellt werden, da der Zielcomputer die Verbindung verweigerte'))" type="AssertionError"/>
</testcase>
</testsuite>
